src/contribute/contribution-guide.md

```diff
@@ -308,7 +308,7 @@ From another local branch, run:
 The community will close stale pull requests in order to keep the project healthy. A pull request becomes stale after its author fails to respond to actionable comments for 60 days.  Author of a closed pull request is welcome to reopen the same pull request again in the future. The associated JIRAs will be unassigned from the author but will stay open.
 
 ## Commit (committers only)
-Once the code has been peer reviewed by a committer, the next step is for the committer to merge it into the [authoritative Apache repository](https://git-wip-us.apache.org/repos/asf/beam.git), not the read-only GitHub mirror. (In the case that the author is also a committer, it is acceptable for either the author of the change or committer who reviewed the change to do the merge. Just be explicit about whose job it is!)
+Once the code has been peer reviewed by a committer, the next step is for the committer to merge it into the [authoritative Apache repository](https://gitbox.apache.org/repos/asf/beam.git), not the read-only GitHub mirror. (In the case that the author is also a committer, it is acceptable for either the author of the change or committer who reviewed the change to do the merge. Just be explicit about whose job it is!)
```

Comment:
is the following statement still true? we(committers) actually have `push` rights in Github, right? 

> the next step is for the committer to merge it into the [authoritative Apache repository](https://gitbox.apache.org/repos/asf/beam.git), not the read-only GitHub mirror.

Reply:
You are right this can be improved, however I rushed to merge this PR because the dead links are blocking the validation of all the other PRs in the website.
WDYT of fixing this @jbonofre

Reply:
You can push on both github and gitbox (they are sync together).

Reply:
But you are right, it could be updated.